css/style.css

```diff
@@ -0,0 +1,19 @@
+body{
+  text-align: center;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Helvetica Neue', Arial, sans-serif;
+  font-size: 1.6rem;
+  line-height: 1.5;
+  color: #333;
+}
+#main{
+  margin: 30px 8px;
+  padding: 15px;
+  border-radius: 5px;
+  background: #eee;
+}
+img{
```

Comment:
Consider putting a blank space between the word and the curly braces. It was one error that the linter is showing.
And a blank line is necessary between the previous curly braces and the new declaration